pkg/validators/ocm_client.go

```diff
@@ -0,0 +1,39 @@
+package validators
```

Comment:
Any reason this is not in `types` package? We risk circular dependencies as `package validators` is imported and imports stuff, where `types` is import-only package.